gravitee-apim-console-webui/src/management/api/policy-studio/debug/components/policy-studio-debug-timeline-card/policy-studio-debug-timeline-card.component.html

```diff
@@ -37,7 +37,7 @@
   </div>
   <div class="policy-studio-debug-timeline-card__content">
     <div *ngIf="timelineCardVM.headerLabel" class="policy-studio-debug-timeline-card__content__header-label">
-      {{ timelineCardVM.headerLabel }}
+      {{ timelineCardVM.stage | titlecase }} > {{ timelineCardVM.headerLabel }}
```

Comment:
It's little strange to have something with specific name as `headerLabel` and a condition to display this one. But what is displayed as a header label is a concatenation with `stage`.
It would be better to rename `headerLabel`as `flowName` or to made the concatenation when the headerLabel is assigned

Reply:
done in the component